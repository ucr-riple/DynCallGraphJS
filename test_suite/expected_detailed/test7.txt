{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test7.js:8:19:8:27)': [
        'foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test7.js:2:1:4:2)'
    ],
    'foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test7.js:3:10:3:15)': [
        'bar (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test7.js:5:1:7:2)'
    ]
}