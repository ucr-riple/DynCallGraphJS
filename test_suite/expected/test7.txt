{
  '(test7.js:8:19:8:27)': [ '(test7.js:2:1:4:2)' ],
  '(test7.js:3:10:3:15)': [ '(test7.js:5:1:7:2)' ]
}